cns/networkcontainers/networkcontainers_windows.go

```diff
@@ -160,6 +161,8 @@ func createOrUpdateWithOperation(adapterName string, ipConfig cns.IPConfiguratio
 		"true",
 		"/weakhostreceive",
 		"true",
+		"/renamewithnetsh",
+		fmt.Sprintf("%t", configuration.CnsConfig.EnableRenameWithNetSh),
```

Comment:
Should not be consumed from a global. It should be passed as a parameter (like `setWeakHost`)